### PR TITLE
[indy-sdk] do not depend on node type

### PR DESCRIPTION
**Pull request by TimoGlastra**
The package can be used outside of node environments, thus removing the dependency on `@types/node`

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [-] Provide a URL to documentation or source code which provides context for the suggested changes: none, shouldn't have depended on NodeJS types
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by karimStekelenburg — APPROVED** [review]

File: types/indy-sdk/tslint.json [hunk 1]
```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false
```
Comment (elibarzilay) [inline types/indy-sdk/tslint.json:4]: Why is this needed?

Comment (TimoGlastra) [reply types/indy-sdk/tslint.json]: The ci was giving me errors and this was what I could find to fix it. However I've made some changes afterwards, so maybe it will also work without it. I've reverted the change to see if the pipeline will pass

Comment (TimoGlastra) [reply types/indy-sdk/tslint.json]: Seems to work. Thanks!

File: types/indy-sdk/tsconfig.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "ESNext.BigInt"
```
Comment (elibarzilay) [inline types/indy-sdk/tsconfig.json:6]: Sorry, I forgot to ask about this earlier too: is it needed?

Comment (TimoGlastra) [reply types/indy-sdk/tsconfig.json:6]: Yes. the `buffer` library relies on it

Comment (elibarzilay) [reply types/indy-sdk/tsconfig.json:6]: That's not good, since it's basically requires your consumers to use `BigInt` too.  It seems to be a problem in `buffer`, so I made [this PR](https://github.com/feross/buffer/pull/299), but no idea how live is that project so making it possible to merge this in the meantime.

Comment (TimoGlastra) [reply types/indy-sdk/tsconfig.json:6]: I'll merge for now, but will make an additional PR once your PR is merged.

Thank you!

**Comment by TimoGlastra** [comment]
Ready to merge
